Resources/config/admin.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <!-- ARTICLE -->
+        <parameter key="sonata.article.admin.article.class">Sonata\ArticleBundle\Admin\ArticleAdmin</parameter>
+        <parameter key="sonata.article.admin.article.controller">SonataArticleBundle:ArticleAdmin</parameter>
```

Comment:
I use parameters to provide default values, but you're right, I think providing default values can be done with configuration too. So to sum up, having no parameters at all makes sense : it forces us to make sure that everything that should be configurable is configurable "nicely", thanks to the configuration, instead of having the end user fiddle with container parameters directly.